plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -1232,6 +1234,48 @@ private static function should_enable_hpos_for_new_shop() {
 		return apply_filters( 'woocommerce_enable_hpos_by_default_for_new_shops', true );
 	}
 
+	/**
+	 * Get the wc_order_stats table schema.
+	 *
+	 * @since 10.4.0
+	 * @param string $collate Database collation.
+	 * @return string SQL schema for wc_order_stats table.
+	 */
+	private static function get_order_stats_table_schema( $collate ) {
+		global $wpdb;
+
+		$should_have_fulfillment_column = self::is_new_install();
```

Comment:
One additional question around this, should we also limit this by the fact the fulfillment feature is enabled? Or is it pretty certain that this will become the default, so having the column there won't hurt to much.
I don't currently have a strong preference btw.